### PR TITLE
Release v1.0.6

### PR DESCRIPTION
**Pull request by bigfishdesign13**
### Updates

- Updates the following components to use React's `forwardRef` wrapper function to pass along refs to the correct DOM element. This is for third-party packages and components such as Chakra's `Tooltip` to work correctly.
  - `Accordion`, `Breadcrumbs`, `Button`, `ButtonGroup`, `Card`, `ComponentWrapper`, `FieldSet`, `Form`, `SimpleGrid`, `Heading`, `HelperErrorText`, `Hero`, `HorizontalRule`, `Icon`, `Image`, `Label`, `List`, `Logo`, `Modal`, `Notification`, `Pagination`, `ProgressIndicator`, `SearchBar`, `SkeletonLoader`, `SkipNavigation`, `Slider`, `StatusBadge`, `StructuredContent`, `Table`, `Tabs`, `Template`, `TemplateAppContainer`, `Text`, `VideoPlayer`.
- Updates `padding-left` and `padding-right` style properties to `padding-start` and `padding-end`, respectively. Updates `margin-left` and `margin-right` style properties to `margin-start` and `margin-end`, respectively. These changes apply to inline CSS, shorthand Chakra style prop syntax, and Chakra theme file syntax. Note that `margin-start/end` and `padding-start/end` get converted to `margin-inline-start/end` and `padding-inline-start/end` respectively by Chakra UI.
- Updates the `Icon` component to include variants for legacy versions of `Account` (filled and unfilled), `Facebook`, `Instagram`, `Twitter` and `YouTube`.
- Updates the `Icon` component to include variants for `Warning` (filled and oultine).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,17 +8,33 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
...
+- Updates `padding-left` and `padding-right` style properties to `padding-start` and `padding-end`, respectively. Updates `margin-left` and `margin-right` style properties to `margin-start` and `margin-end`, respectively. These changes apply to inline CSS, shorthand Chakra style prop syntax, and Chakra theme file syntax. Note that `margin-start/end` and `padding-start/end` get converted to `margin-inline-start/end` and `padding-inline-start/end` respectively by Chakra UI.
+- Updates the `Icon` component to include variants for legacy versions of `Account` (filled and unfilled), `Facebook`, `Instagram`, `Twitter` and `YouTube`.
+- Updates the `Icon` component to include variants for `Warning` (filled and oultine).
+
 ## 1.0.5 (July 7, 2022)
 
 ### Adds
 
 - Exports `ColorCard`.
+- Adds the `isLazy` prop to the `Image` component which is set to `false` by default.
```
Comment (EdwinGuzman) [inline CHANGELOG.md]: This should move up to the "Prerelease" section.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,17 +8,33 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
...
 ### Updates
 
 - Updates the `Icon` component to include variants for `actionCheckCircleFilled`, `actionRegistration`, `decorativeEnvelope`, `decorativeLibraryCard` and `decorativeShoppingBag`.
 - Updates the docs for the `Colors` Style Guide to include additional data related to contrast and WCAG compliance.
 - Updates the `Checkbox` and `Radio` components to use `ui.gray.dark` for the default border color.
+- Updates Storybook packages.
+
+### Removes
+
+- Removes the packages `@mdx-js/react` and `babel-loader` since they are not needed by Storybook.
```
Comment (EdwinGuzman) [inline CHANGELOG.md]: This should move up to the "Prerelease" section.

**Review by EdwinGuzman — CHANGES_REQUESTED** [review]
Just a minor update to the changelog

File: CHANGELOG.md (hunk 1 above)
Comment (bigfishdesign13) [reply CHANGELOG.md]: I see it’s in the wrong place, but shouldn't it actually go under the `1.0.6` heading?

File: CHANGELOG.md (hunk 2 above)
Comment (bigfishdesign13) [reply CHANGELOG.md]: Same for this.  Wasn’t this part of [DSD-1051](https://jira.nypl.org/browse/DSD-1051)?

Comment (EdwinGuzman) [reply CHANGELOG.md]: Right, I meant under 1.0.6 instead of Prerelease.

File: CHANGELOG.md (hunk 1 above)
Comment (EdwinGuzman) [reply CHANGELOG.md]: Yeah, that's what I meant.

**Review by EdwinGuzman — APPROVED** [review]

**Pull request merged**